components/dialogs/dialogs.js

```diff
@@ -132,9 +132,9 @@ class YesNoMuteDialogViewModel extends PromptDialogViewModel {
     this.result = ko.observable(false);
     const self = this;
     this.alternatives([
-      { label: 'Yes', primary: true, taId: 'yes', click() { self.result(true); self.close(); } },
-      { label: 'Yes and mute for awhile', primary: false, taId: 'mute', click() { self.result("mute"); self.close() } },
-      { label: 'No', primary: false, taId: 'no', click() { self.result(false); self.close(); } }
+      { label: 'Yes', style: 'primary', taId: 'yes', click() { self.result(true); self.close(); } },
+      { label: 'Yes and mute for awhile', style: 'mute', click() { self.result("mute"); self.close() } },
```

Comment:
The default `btn btn-default` classes changed to only `btn` [here](https://github.com/FredrikNoren/ungit/pull/1318/files#diff-474b93f945d2a9ad29f677e433974d0eR12) which now looks like this:

Before:
![image](https://user-images.githubusercontent.com/4009570/80237062-5dfa0100-865c-11ea-8ee2-32c99948846f.png)

After:
![image](https://user-images.githubusercontent.com/4009570/80237070-65210f00-865c-11ea-8330-2b593f3ac66e.png)

Was this intended?

Reply:
Good catch.

I didn't realize `btn-muted` was not a style class.
Changed it now to separate style classes from action/testing ids.